### ISSUE TITLE
External images must be loaded from Tutanota's servers

### ISSUE THREAD

**Issue by x80486** [tutao/tutanota#1279]
> Not actually a bug on its own, but something you should try to fix while loading external resources.

I understand remote images are blocked to protect user's privacy...and that's a great start: by blocking any images (or any other content) that wasn't embedded/sent with the email by default, so one must perform an action to load the remote content in order to view it.

Since this is not going to stop, and every time we load a remote image/resource, companies often want to be notified (and who knows what else) that the recipient accessed the resource in question, I think it's wise to implement something that, every time a user perform such action (read: load the remote content), that should be performed, transparently, by Tutanota's servers on user's behalf (potentially with all the bells and whistles one can ask for), so the image/resource host still doesn't know the actual user's internet address, browser and what not.

**Comment by charlag** [comment]
Well, that sounds like a proxy or a VPN, doesn't it?
We may revisit later but we don't have any immediate plans (or spare hands) to implement this right now.

**Comment by snaggen** [comment]
This kind of proxy solution is kind of toothless , the only thing needed to keep tracking users through the proxy is to make the image url unique for each user. 
I think google added a proxy solution, which solved this problem by always downloading the images and rewriting the urls to point to the local cached image. However, that way of solving the issue is not possible with end to end encryption.  
So, in short, a proxy doesn't really fix anything and there is not much that can be done.
